### ISSUE TITLE
Error while decoding draco3d encoded file (using both cli and core)

### ISSUE THREAD

**Issue by timknip** [donmccurdy/glTF-Transform#280]
**Describe the bug**
Some draco encoded files fail to decode
[10077871_011.zip](https://github.com/donmccurdy/glTF-Transform/files/6727941/10077871_011.zip)

**To Reproduce**
1. unzip attached file
2. run `gltf-transform inspect 10077871_011.glb`
```
 METADATA
 ────────────────────────────────────────────
┌────────────────────┬──────────────────────────────────────────────────────┐
│ key                │ value                                                │
├────────────────────┼──────────────────────────────────────────────────────┤
│ version            │ 2.0                                                  │
├────────────────────┼──────────────────────────────────────────────────────┤
│ generator          │ Gestaltor 2021.2.0 Spelt Professional (dev-66625ca*) │
├────────────────────┼──────────────────────────────────────────────────────┤
│ extensionsUsed     │ KHR_draco_mesh_compression, KHR_materials_variants   │
├────────────────────┼──────────────────────────────────────────────────────┤
│ extensionsRequired │ KHR_draco_mesh_compression                           │
└────────────────────┴──────────────────────────────────────────────────────┘


warn: Unable to parse document.

error: Cannot read property 'setArray' of undefined
```
3.  run code using @gltf-transform/core@0.11.4, @gltf-transform/extensions@0.11.4 and draco3dgltf@1.4.1:

```
async function bug () {
    const filename = '10077871_011.glb'
    const decoder = await draco3d.createDecoderModule();
    const encoder = await draco3d.createEncoderModule();
    const io = new NodeIO()
        .registerExtensions([DracoMeshCompression, MaterialsVariants])
        .registerDependencies({
            'draco3d.decoder': decoder,
            'draco3d.encoder': encoder
        });
    const doc = io.read(filename);
}

(async () => {
   await bug()
})()

TypeError: Cannot read property 'setArray' of undefined
    at o.s.preread (D:\github\floorplanner\tmp\fp.thirdparty.models\packages\gltf\node_modules\@gltf-transform\extensions\dist\extensions.js:1:11289)
    at D:\github\floorplanner\tmp\fp.thirdparty.models\packages\gltf\node_modules\@gltf-transform\core\dist\core.js:1:55234
    at Array.forEach (<anonymous>)
    at Function.t.read (D:\github\floorplanner\tmp\fp.thirdparty.models\packages\gltf\node_modules\@gltf-transform\core\dist\core.js:1:55205)
    at n.r.read (D:\github\floorplanner\tmp\fp.thirdparty.models\packages\gltf\node_modules\@gltf-transform\core\dist\core.js:1:75428)
```
**Expected behavior**
no errors thrown

**Versions:**
 - Version: @gltf-transform/cli@v0.11.4 with draco3dgltf@1.4.1
 - Environment: Node v12.19.1, Windows 10 x64

**Additional context**
Attached file decodes and displays fine in:
-  https://gltf-viewer.donmccurdy.com/
-  https://sandbox.babylonjs.com/

**Comment by donmccurdy** [comment]
Interesting case here... I wasn't aware it was possible to create a Draco-compressed mesh without including primitive indices. I'm not quite sure if Gestaltor is doing the right thing, or if my reading of the spec and Draco API is incorrect. I can see why the viewers would be OK in this case, but there are some other edge cases to consider if the Draco library merges vertices. I've filed an issue to ask on https://github.com/google/draco/issues/720.

**Comment by donmccurdy** [comment]
Thanks for letting me know, this should be fixed in v0.11.5. 👍

**Comment by timknip** [comment]
No problem ;-) Thanks for the interesting read and the great tool.
